src/NCronJob/Registry/DependingJobRegistry.cs

```diff
@@ -2,25 +2,28 @@
 
 namespace NCronJob;
 
+// TODO: This might eventually be pushed down intio JobRegistry
```

Comment:
Do we really need this `DependingJobRegistry` type?
Shouldn't this be directly handled by the `JobRegistry`?

Reply:
That is a good point. The Depending Job Registry might be a piece I needed for the initial draft for building job dependencies. I refactored some of the code quite heavily (not always for the better). So yes, there might be things that aren0t needed anymore! Really nice that you spotted that.

Reply:
The type has been dropped entirely